tmva/pymva/inc/TMVA/RModelParser_PyTorch.h

```diff
@@ -0,0 +1,89 @@
+// @(#)root/tmva/pymva $Id$
+// Author: Sanjiban Sengupta, 2021
+
+/**********************************************************************************
+ * Project: TMVA - a Root-integrated toolkit for multivariate data analysis       *
+ * Package: TMVA                                                                  *
+ * Web    : http://tmva.sourceforge.net                                           *
+ *                                                                                *
+ * Description:                                                                   *
+ *      Functionality for parsing a saved PyTorch .PT model into RModel object    *
+ *                                                                                *
+ * Authors (alphabetical):                                                        *
+ *      Sanjiban Sengupta <sanjiban.sg@gmail.com>                                 *
+ *                                                                                *
+ * Copyright (c) 2021:                                                            *
+ *      CERN, Switzerland                                                         *
+ *                                                                                *
+ *                                                                                *
+ * Redistribution and use in source and binary forms, with or without             *
+ * modification, are permitted according to the terms listed in LICENSE           *
+ * (http://tmva.sourceforge.net/LICENSE)                                          *
+ **********************************************************************************/
+
+
+#ifndef TMVA_SOFIE_RMODELPARSER_PYTORCH
+#define TMVA_SOFIE_RMODELPARSER_PYTORCH
+
+#include <Python.h>
+
+#define NPY_NO_DEPRECATED_API NPY_1_7_API_VERSION
+#include <numpy/arrayobject.h>
+
+#include "TMVA/RModel.hxx"
+#include "TMVA/SOFIE_common.hxx"
+#include "TMVA/Types.h"
+#include "TMVA/OperatorList.hxx"
+
+#include "TMVA/PyMethodBase.h"
+
+#include "Rtypes.h"
+#include "TString.h"
+
+
+namespace TMVA{
+namespace Experimental{
+namespace SOFIE{
+namespace PyTorch{
+
+// Referencing Python utility functions present in PyMethodBase
+static void(& PyRunString)(TString, PyObject*, PyObject*) = PyMethodBase::PyRunString;
+static const char*(& PyStringAsString)(PyObject*) = PyMethodBase::PyStringAsString;
+
```

Comment:
Can you please remove INTERNAL functions in the .cxx file as it has been done for the Keras converter

Reply:
Moved the INTERNAL functions to the Source file